python/dgl/mock_sparse/cpp_interface.py

```diff
@@ -0,0 +1,63 @@
+""" DGL sparse library C++ binding"""
+import os
+import torch
+from typing import List, Optional, Tuple
+
+package_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
+so_path = os.path.join(
+    package_path, "../tensoradapter/pytorch/libdgl_sparse.so"
+)
+torch.classes.load_library(so_path)
+
+_C_SparseMatrix = torch.classes.dgl_sparse.SparseMatrix
+
+
+# TODO Add docstring
```

Comment:
Just copy-paste from the mock-up library.

Reply:
Done.